subprojects/core/src/integTest/groovy/org/gradle/api/tasks/MissingTaskDependenciesIntegrationTest.groovy

```diff
@@ -407,6 +407,43 @@ class MissingTaskDependenciesIntegrationTest extends AbstractIntegrationSpec imp
         executedAndNotSkipped(":produceInBuild", ":showSources")
     }
 
+    @Issue("https://github.com/gradle/gradle/issues/16980")
+    def "missing dependency detection takes ** excludes for non-existing files into account"() {
```

Comment:
I'm having trouble understanding the problem from this title. How about:

```suggestion
    def "missing task dependency detection does not trigger for excluded non-existing files"() {
```

Reply:
I think the original name is better.